script/prerender.tsx

```diff
@@ -47,22 +51,31 @@ async function render() {
     }, null, 2));
   }
 
-  // book version is currently ignored, and will be until we rejigger the book content
-  // routing to preserve long id and version data on navigation
-  async function renderContentPage(bookId: string, _bookVersion: string, pageId: string) {
-    const book = await archiveLoader.book(bookId);
-    const page = await archiveLoader.page(bookId, pageId);
+  async function renderContentPage(bookId: string, bookVersion: string, pageId: string) {
+    const book = await archiveLoader.book(`${bookId}@${bookVersion}`);
```

Comment:
Why concatenate the string here rather than in `archiveLoader` ? It seems archiveLoader is already doing string concatenation for page-in-a-book.

By stripping the version out in multiple places (& keeping the same type for `yqV9q0HH` and `yqV9q0HH@2`) it is difficult to see when a version is being used/needed and when it is not.

Reply:
i'll make a change so that the book version gets passed into archiveloader explicitly, i think that'll help